sympy/solvers/ode/ode.py

```diff
@@ -1827,12 +1967,16 @@ def _sympify(eq):
     for i, fi in enumerate(eq):
         if isinstance(fi, Equality):
             eq[i] = fi.lhs - fi.rhs
+
+    # match = neq_nth_linear_constant_coeff_match(eq, funcs)
+    # if match is not None:
+    #     return match
```

Comment:
It doesn't look like the function is being used.

Reply:
Yes, I will update this in the next commit.